dissect/target/plugins/apps/webserver/iis.py

```diff
@@ -84,11 +93,23 @@ def get_log_dirs(self) -> list[tuple[str, Path]]:
         except (ElementTree.ParseError, DissectFileNotFoundError) as e:
             self.target.log.warning(f"Error while parsing {self.config}: {e}")
 
+        for log_path in self.DEFAULT_LOG_PATHS:
+            for log_file in self.target.fs.glob(log_path):
+                try:
+                    log_dir = self.target.fs.path(log_file).parents[1]
+                    if (
+                        "auto",
+                        log_dir,
+                    ) not in log_paths:
+                        log_paths.append(("auto", log_dir))
```

Comment:
Maybe let `log_paths` be a set instead of a list, such that you don't need this if-statement

Reply:
ok, good one